client/src/pages/CalendarPage.js

```diff
@@ -150,7 +242,8 @@ export default function CalendarPage() {
                 </Grid>
                 <Grid item container direction="row">
                     <Grid item className={classes.colMid} sm={8}>
-                        <CalendarWidget minDate={minDate} />
+                        <CalendarWidget minDate={minDate} handleClickDay={handleClickDay}
+                            selectedDay={selectedDay} />
```

Comment:
line 257 replace with selectedDay instead of fixed Feb 12

Reply:
Yeah, I've already done that in the next branch

Reply:
ok nice nice